objects/activities/authentication_related_activity.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
All of these new common attribute files (or "base class files" as I call them) are included either directly or indirectly by events and obects. Is it appropriate therefore that they live under objects/* instead of includes/*?